src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -446,13 +446,13 @@ namespace BaseboardElectric {
             auto &ZoneEqSizing = state.dataSize->ZoneEqSizing(state.dataSize->CurZoneEqNum);
             auto &baseboard = state.dataBaseboardElectric->baseboards(BaseboardNum);
 
-            std::string CompType = baseboard.EquipType;
-            std::string CompName = baseboard.EquipName;
+            std::string_view const CompType = baseboard.EquipType;
+            std::string_view const CompName = baseboard.EquipName;
             state.dataSize->DataFracOfAutosizedHeatingCapacity = 1.0;
             state.dataSize->DataZoneNumber = baseboard.ZonePtr;
             int SizingMethod = DataHVACGlobals::HeatingCapacitySizing;
             int FieldNum = 1;
-            std::string SizingString = baseboard.FieldNames(FieldNum) + " [W]";
+            std::string_view const SizingString = baseboard.FieldNames(FieldNum) + " [W]";
```

Comment:
For some reason this cannot be a string_view. There is a linux warning:

    object backing the pointer will be destroyed at the end of the full-expression [-Wdangling-gsl]

and a strange diff:

    Device id: x86_64-MacOS-10.17-clang-13.0.0Comparing 
    `/Users/commbldg/ci/clone_baseline/build/testfiles/1ZoneDataCenterCRAC_wApproachTemp` with 
    `/Users/commbldg/ci/clone_branch/build/testfiles/1ZoneDataCenterCRAC_wApproachTemp`Traceback (most recent 
    call last):File "/Users/commbldg/ci/clone_regressions/energyplus_regressions/diffs/ci_compare_script.py", line 375, 
    inmain_function(File "/Users/commbldg/ci/clone_regressions/energyplus_regressions/diffs/ci_compare_script.py", 
    line 74, in main_functionentry, message = runner.process_diffs_for_one_case(File 
    "/Users/commbldg/ci/clone_regressions/energyplus_regressions/diffs/../../energyplus_regressions/runtests.py", line 
    958, in process_diffs_for_one_casethis_entry.add_text_differences(TextDifferences(SuiteRunner.diff_text_files(File 
    "/Users/commbldg/ci/clone_regressions/energyplus_regressions/diffs/../../energyplus_regressions/runtests.py", line 
    536, in diff_text_filestxt2 = f_txt_2.readlines()File 
    "/usr/local/Cellar/python@3.9/3.9.14/Frameworks/Python.framework/Versions/3.9/lib/python3.9/codecs.py", line 
    322, in decode(result, consumed) = self._buffer_decode(data, self.errors, final)UnicodeDecodeError: 'utf-8' codec 
    can't decode byte 0x8b in position 4507: invalid start byte